google-cloud-billing-budgets-v1beta1/synth.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool as s
+import synthtool.gcp as gcp
+import synthtool.languages.ruby as ruby
+import logging
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICMicrogenerator()
+library = gapic.ruby_library(
+    "billing/budgets", "v1beta1",
+    extra_proto_files=["google/cloud/common_resources.proto"],
+    generator_args={
+        "ruby-cloud-gem-name": "google-cloud-billing-budgets-v1beta1",
+        "ruby-cloud-title": "Billing Budgets V1beta1",
+        "ruby-cloud-description": "Provides methods to view, create, and manage Cloud Billing budgets programmatically at scale.",
+        "ruby-cloud-env-prefix": "BUDGETS",
```

Comment:
Would it be better to include the product "family" name with `BILLING_ BUDGETS`? For example, we have `BIGQUERY_CONNECTION`, `BIGQUERY_RESERVATION` and `BIGQUERY_STORAGE` in the BigQuery family.

Reply:
Yeah, I think you're right, `BILLING_BUDGETS` is better. Changed. (We're not totally consistent though. bigquery-data_transfer is `DATA_TRANSFER`...)

Reply:
> We're not totally consistent though. bigquery-data_transfer is DATA_TRANSFER

I know :(